install/victoriametrics-install.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: Denys Holius https://github.com/denisgolius
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://victoriametrics.com/
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt install -y \
+          curl \
+          sudo \
+          mc
+
+msg_ok "Installed Dependencies"
+
+msg_info "Installing VictoriaMetrics"
+RELEASE=$(curl -s https://api.github.com/repos/VictoriaMetrics/VictoriaMetrics/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+mkdir -p /etc/victoriametrics
+mkdir -p /var/lib/victoriametrics
+groupadd -r victoriametrics
+useradd -g victoriametrics -d /var/lib/victoriametrics -s /sbin/nologin --system victoriametrics
+chown -R victoriametrics:victoriametrics /var/lib/victoria-metrics
+wget -q https://github.com/VictoriaMetrics/VictoriaMetrics/releases/download/v${RELEASE}/victoria-metrics-linux-amd64-${RELEASE}.tar.gz
+tar -xf victoria-metrics-linux-amd64-${RELEASE}.tar.gz
+chmod +x /usr/bin/victoria-metrics-prod
+chown root:root /usr/bin/victoria-metrics-prod
+
+cat <<END >/etc/victoriametrics/scrape.yml
+# Scrape config example
+#
+scrape_configs:
+  - job_name: self_scrape
+    scrape_interval: 10s
+    static_configs:
+      - targets: ['127.0.0.1:8428'] 
+END
+
+mv victoria-metrics-prod /usr/local/bin/
```

Comment:
```suggestion
mv victoria-metrics-prod /opt/vistoriamerics
```
Cant it exist under /opt? This is the standard Path for all apllictions if possible. You would have to change some other paths too when this is possble.

Reply:
I don't understand you point here. Most of software that you already have in the repository use /usr/local/bin so why in this PR I will need to use opt folder? Do you have best practices or requirements with step-to-step for adding new software?

Reply:
PS: As you may saw I've used Prometheus source files for my PR, so you plan to rewrite all scripts and migrate from `/usr/local/bin` to `/opt`?

Reply:
A small amount of software should be in /usr. The majority is in /opt so that everyone can find it again.
If this is not possible, it remains in /usr, but if it is possible to run it via /opt, this is the better solution

Reply:
the migration is to hard, because all exist scripts broken after this. 
But at new scripts its easy :P